INTERPRETER.s

```diff
@@ -1337,7 +1337,7 @@ UPPOS		XCH	L		# SAVE DECREMENTED UPPER PART.
 		TS	A		# SKIPS ON OVERFLOW
 		TCF	+2
 		INCR	L		# RESTORE UPPER TO ORIGINAL VALUE
```

Comment:
Another non-spotted typo.

`ORIGINAL` should be `ROIGINAL` - 10/10 1969 typo. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1148.jpg)